app/src/main/java/org/mozilla/fenix/tabtray/TabTrayDialogFragment.kt

```diff
@@ -14,13 +14,19 @@ import kotlinx.android.synthetic.main.component_tabstray.view.*
 import kotlinx.android.synthetic.main.fragment_tab_tray_dialog.*
 import kotlinx.android.synthetic.main.fragment_tab_tray_dialog.view.*
 import mozilla.components.concept.tabstray.Tab
+import mozilla.components.lib.state.ext.consumeFrom
 import org.mozilla.fenix.HomeActivity
 import org.mozilla.fenix.R
+import org.mozilla.fenix.ext.requireComponents
 
 class TabTrayDialogFragment : AppCompatDialogFragment(), TabTrayInteractor {
+
     interface Interactor {
         fun onTabSelected(tab: Tab)
         fun onNewTabTapped(private: Boolean)
+        fun onShareTabsClicked(private: Boolean)
+        fun onSaveToCollectionClicked(private: Boolean)
```

Comment:
I would remove the private flag here as we don't allow to save private tabs to collections